docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -1,44 +1,62 @@
 import importlib
 
+import numpy
+
+blocklist = [
+    "test",
+    "add_docstring",
+    "abs",
+    "add_newdoc",
+    "add_newdoc_ufunc",
+    "alen",
+    "alltrue",
+    "bitwise_not",
```

Comment:
why is this excluded? we did implement `bitwise_not`.

Reply:
because these functions are not part of numpy official API: https://numpy.org/doc/stable/search.html?q=bitwise_not

Reply:
If we don't filter them out, there will be no valid link for the numpy API for them